src/SIL.XForge.Scripture/locales.json

```diff
@@ -44,7 +44,7 @@
     "englishName": "Portuguese",
     "tags": ["pt-BR"],
     "production": true,
-    "helps": "pt_BR"
+    "helps": "pt-br"
```

Comment:
Docusaurus is picky and inflexible about the locales. Right now a redirect is in place on the help site to make `/pt_BR` work, but we can simplify the help site by linking directly to `pt-br`.